lisa/tools/ethtool.py

```diff
@@ -131,25 +137,32 @@ class DeviceFeatures:
         r"^[\s]*(?P<name>.*):(?P<value>.*?)?$", re.MULTILINE
     )
 
-    def __init__(self, interface: str, device_feature_raw: str) -> None:
-        self._parse_feature_info(interface, device_feature_raw)
-
-    def _parse_feature_info(self, interface: str, raw_str: str) -> None:
-        matched_features_info = self._feature_info_pattern.search(raw_str)
+    def __init__(self, interface: str, enabled_features: List[str]) -> None:
+        self.device_name = interface
+        self.enabled_features = enabled_features
+
+    @classmethod
+    def from_raw_feature(
```

Comment:
It looks work for Linux only. So it should be moved into Linux tool.

Reply:
Yes. It is specific to Linux. I will move this logic into ethtool class for linux